docs/docs/tutorial/chapter2/getting-dynamic.md

```diff
@@ -42,6 +45,31 @@ model Post {
 // highlight-end
 ```
 
+</TabItem>
+<TabItem value="mongo" label="MongoDB">
+
+```Javascript
```

Comment:
It looks like docusaurus doesn't recognize "Javascript". Lowercasing it again should bring back syntax highlighting
```suggestion
```javascript
```